src/MinimalDelegation.sol

```diff
@@ -132,29 +137,15 @@ contract MinimalDelegation is
         (bytes32 keyHash, bytes memory signature, bytes memory hookData) =
             abi.decode(userOp.signature, (bytes32, bytes, bytes));
 
-        /// The userOpHash does not need to be safe hashed with _hashTypedData, as the EntryPoint will always call the sender contract of the UserOperation for validation.
-        /// It is possible that the signature is a wrapped signature, so any supported key can be used to validate the signature.
-        /// This is because the signature field is not defined by the protocol, but by the account implementation. See https://eips.ethereum.org/EIPS/eip-4337#definitions
+        /// The userOpHash does not need to be made replay-safe, as the EntryPoint will always call the sender contract of the UserOperation for validation.
         Key memory key = getKey(keyHash);
         bool isValid = key.verify(userOpHash, signature);
 
-        // If signature verification failed, return failure immediately WITHOUT expiry as it cannot be trusted
-        if (!isValid) {
-            return SIG_VALIDATION_FAILED;
-        }
-
         Settings settings = getKeySettings(keyHash);
```

Comment:
Add inline comment about this hook must revert to fail in validation

Reply:
Do you think sufficient to put that in IHookValidation? Thats where I updated that behaviour for now

Reply:
ah yeah i think so